apps/angular-example-counter/src/app/types.ts

```diff
@@ -1,7 +1,8 @@
-import { Subject, Observable } from 'rxjs';
+import { Subject } from 'rxjs';
+import { StoreValue } from '@rx-store/core';
 
 /** Our context value, which contains subjects & observables */
-export interface RootAppStore {
+export interface RootAppStore extends StoreValue {
```

Comment:
nit: would consider a rename to `AppStoreValue`?

Reply:
addressed in 8faab1f